torchci/lib/fetchHud.ts

```diff
@@ -24,13 +24,23 @@ export default async function fetchHud(params: HudParams): Promise<{
             type: "int",
             value: params.page.toString(),
           },
+          {
+            name: "owner",
+            type: "string",
+            value: params.repoOwner,
+          },
+          {
+            name: "repo",
+            type: "string",
+            value: params.repoName,
+          },
```

Comment:
I think something that was not super intuitive right away was that these changes are based on actual query changes as well. As we onboard people to maintain torchci, should we add comments here to explain or should we start a general README?

Reply:
What part was not clear? That the bump in query version introduced new parameters? Or that there are queries stored in the rockset backend separately from the rest of the code?